app/controllers/article_controller.rb

```diff
@@ -47,8 +47,13 @@ def list
 
   def delete
     if @article.link_list.empty? && @article.target_article_links.empty?
-      @article.destroy
-      redirect_to collection_subjects_path(@collection.owner, @collection)
+      if @article.created_by == current_user
```

Comment:
Lets add an OR clause here so that the collection owner can also delete articles, even if they did not create the project.  (I think the relevant method is `like_owner?`)